source/reference/minio-mc-admin/mc-admin-service.rst

```diff
@@ -74,30 +72,30 @@ Syntax
 .. mc-cmd:: restart
 
    Restarts MinIO servers.
+   If needed, the command provides some guidance based on status about potential next steps you may need to take on each node.
```

Comment:
I'm wondering how useful this guidance might be for less common situations. I poked around in the diff, seems all the suggestions are basically reboot the node? As worded, this sentence suggests to me more robust diagnosis skills than _did you turn it off and on again?_

Reply:
Reworked to just say that the option may suggest restarting the node.